pkg/sqlite/conversion.go

```diff
@@ -76,7 +76,19 @@ func populateModelChannels(ctx context.Context, pkgs model.Model, q *SQLQuerier)
 	if err != nil {
 		return err
 	}
+	deprecatedValue, err := json.Marshal(registry.DeprecatedProperty{})
+	if err != nil {
+		return err
+	}
+ConvertBundles:
 	for _, bundle := range bundles {
+		for _, prop := range bundle.Properties {
+			if prop.Type == registry.DeprecatedType && prop.Value == string(deprecatedValue) {
```

Comment:
Is it necessary to check `prop.Value`? I *think* we could probably just get away with checking for the `olm.deprecated` property type.